pkgs/by-name/ga/gate/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+let
+  pname = "gate";
+  version = "0.36.7";
+in
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "minekube";
+    repo = "gate";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-WHxpx20O/HuCWqbY4zTxcjyIhW3+FQtTz5sUGAda71g=";
+  };
+
+  vendorHash = "sha256-dswNJQWqN+u/mnpbj9se2j9uEi0ewNTXVlN3WnNbcyg=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = {
+    description = "High-Performance, Low-Memory, Lightweight, Extensible Minecraft Reverse Proxy";
+    longDescription = "Gate is an extensible, high performant & paralleled Minecraft proxy server with scalability, flexibility & excellent server version support - written in Go and ready for the cloud!";
```

Comment:
```suggestion
    longDescription = ''
      Gate is an extensible, high performant & paralleled Minecraft proxy server
      with scalability, flexibility & excellent server version support - written in Go
      and ready for the cloud!
    '';
```

Reply:
Is there a suggested maximum line length? I couldn't find any, but adjusted it to your suggestion now. I guess something around 100?